LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Druid.fi
```

Comment:
Druid.fi -> Druid Ltd

Reply:
https://tietopalvelu.ytj.fi/yritystiedot.aspx?yavain=2403325&tarkiste=EE58791B848CCDA01E3EC459BC0DF8755D3AA641

So it's "Druid Ltd." with the dot.